### PR TITLE
Local postgres

### PR DESCRIPTION
**Pull request by axelssonHakan**

### CODE REVIEW

File: local-postgres.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## First step: Setting up the example
+
+Verify that it´s possible to spin up the docker environment and run the example as described in the [README.md](./README.md).
+
+After running all the steps a sample of the airports table should be visible in the console.
+
+## Second step: Dump the airports table
+
+This section can be skipped and the [airports.tar](./example/postgres-image/airports.tar) be used in the next step
```
Comment (wennmo) [inline local-postgres.md]: The `tar` file should be added to the `.gitattributes` as a binary if we want to check it in. Perhaps better to just document the command for creating a tar file from the csv?

Comment (axelssonHakan) [reply local-postgres.md]: It´s not possible to create directly from the csv. Added it to .gitignore

File: local-postgres.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+1. Get the ID of the database container `docker ps`
+2. Connect to the database container `docker exec -it <CONTAINER ID> sh`
+3. Execute `pg_dump --host localhost --port 5432 --username postgres --format tar --verbose --file airports.tar --table public.airports postgres` in the shell of the database container
+4. Exit that container
+5. Copy the tar from the container with `docker cp <CONTAINER ID>:/airports.tar airports.tar`
+
+## Third step: Setting up local postgres
+
+1. Use [pgAdmin](https://www.pgadmin.org/) to connect to your local (on the development computer). (localhost and port 5432 by default)
+2. Create a new database (rightClick **Create > Databases** under **Servers** and name it **test**)
```
[the remark this answers is not in the document]
Comment (axelssonHakan) [reply local-postgres.md]: Done

File: local-postgres.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+3. RightClick the **test** database and select **Restore...**
```
[the remark this answers is not in the document]
Comment (axelssonHakan) [reply local-postgres.md]: Done

File: local-postgres.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+4. Locale the airports.tar and press **Restore**
```
[the remark this answers is not in the document]
Comment (axelssonHakan) [reply local-postgres.md]: 👍

File: local-postgres.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+- `\c test`  change to the test database
+- `\dt+` list tables
+
+It´s also possible to execute SQL statemets directly (you may need to change to correct database as described above)
+- `SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE='BASE TABLE' AND TABLE_SCHEMA='public';`
+- `SELECT * FROM airports;`
+
+## Using corectl
+
+Instead of using the programatic approach the tool [corectl](https://github.com/qlik-oss/corectl/releases) can be used to desribe the app you want to create.
```
[the remark this answers is not in the document]
Comment (axelssonHakan) [reply local-postgres.md]: Thanks

File: local-postgres.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+script.qvs:
+```
+lib connect to 'postgres-grpc-connector';
+
+Airports:
+sql SELECT rowID,Airport,City,Country,IATACode,ICAOCode,Latitude,Longitude,Altitude,TimeZone,DST,TZ, clock_timestamp() FROM airports ORDER BY Airport;
+```
+
+Let´s use our declarative files to generate the app.
```
[the remark this answers is not in the document]
Comment (axelssonHakan) [reply local-postgres.md]: Changed

**Pull request merged**